core/src/main/scala/ai/lum/odinson/ExtractorEngine.scala

```diff
@@ -1,10 +1,11 @@
 package ai.lum.odinson
 
 import java.nio.file.Path
+
 import org.apache.lucene.analysis.core.WhitespaceAnalyzer
-import org.apache.lucene.document.{ Document => LuceneDocument }
```

Comment:
@kwalcock, @marcovzla : maybe we should add [a formatting hook](https://scalameta.org/scalafmt/) to avoid an intentional diff war?  I'm of the habit of always adding those spaces, but I think it's more important that we try to be consistent.

Reply:
I usually put a space there, but would not have changed that on purpose.  I assume that IntelliJ did it without me noticing, which is why I generally do not like tools taking that initiative.  Sorry I didn't notice.  The goal is to adapt to whatever format is present, whether it is wrong or not ;-)  If our tools are conflicting, though, and fighting each other, a tie-breaker might be necessary.  As far as I know, I'm using default settings in IntelliJ and generally haven't noticed many conflicts of this sort.

Reply:
Ah, gotcha.  No problem anyway (at least not a pressing one!).